### ISSUE TITLE
Support for UFF18

### ISSUE THREAD

**Issue by janheindejong** [ladisk/pyuff#90]
Great tool, love it! 

I saw it doesn't officially support UFF18, but there seems to be code to do so. Any plans to officially support this?

**Comment by jankoslavic** [comment]
dear @janheindejong I invite you to prepare a pull request; you can count on my support.

**Comment by janheindejong** [comment]
Ok, I'll give it a crack

**Comment by jankoslavic** [comment]
Yes, please start from the code available. Try to keep similar logic as is used elsewhere. Add a simple testing as used for other datatypes.

**Comment by jankoslavic** [comment]
Hi @janheindejong any progress?
